### PR TITLE
Fixes #4742 - search.php warning

### LINKED ISSUE #4742: search.php PHP warning
https://community.mybb.com/thread-239134.html

### PR DESCRIPTION
**Pull request by Eldenroot**
Resolves #4742

### CODE REVIEW

File: search.php [hunk 1]
```diff
@@ -980,6 +980,7 @@
 			{
 				$folder .= "new";
 				$folder_label .= $lang->icon_new;
+				$thread['newpostlink'] = get_thread_link($post['thread_lastpost']);
```
Comment (Stefan-MyBB) [inline search.php]: Isn't this link supposed to point to the last post?

Comment (dvz) [reply search.php]: Should then probably be `get_thread_link($post['tid'], 0, "newpost")`

Comment (Eldenroot) [reply search.php]: Fixed, thx!

**Comment by Eldenroot** [comment]
Fixed

**Pull request merged**